documentation/tags.md

```diff
@@ -193,6 +195,8 @@ Naive Bayes
 Natural Language Processing
 Netlify
 Networking
+Neural Networks
+Neurons
```

Comment:
```suggestion
```
This is duplicative of Neural Networks, and is less precise.

Reply:
done